main.go

```diff
@@ -67,6 +68,12 @@ func main() {
 				var file *os.File
 				var err error
 
+				if index == "" {
+					t := time.Now()
```

Comment:
I think we should use a date and time here, because otherwise you are going to run the risk of overwriting an index if you index more than once per day. I'd also suggest using UTC time, `time.Format` and the RFC3339 constant for formatting. It's much easier:
```
t := time.Now().UTC()
index = fmt.Sprintf("%s-%s", "aleph", t.Format(time.RFC3339))
```

Reply:
I did intentionally limit it to a daily index as that is the expected use case. I understand your point and I considered that when working on this. I'll give it a bit more thought and let you know what I'm intending to do here.

Reply:
After offline discussion, we will use the timestamp as suggested.